cfn-resources/database-user/mongodb-atlas-databaseuser.json

```diff
@@ -131,7 +132,8 @@
       },
       "minItems": 1,
       "type": "array",
-      "uniqueItems": true
+      "uniqueItems": true,
+      "insertionOrder": false
```

Comment:
not strongly against this but I'd probably just remove the cfn generate warnings at the moment, using the default value so we don't change the current behavior

Reply:
as this is the only change that requires a release, I'd probably revert this change so we only change README and examples and we don't need a release